src/main/java/seedu/address/model/person/Name.java

```diff
@@ -16,7 +16,7 @@
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "^[^\\s][\\p{Alnum}\\s]+$";
+    public static final String VALIDATION_REGEX = "^[\\p{L} .'-]+$";
```

Comment:
Since this regex will not trim whitespaces, are we still going to keep it for v1.2?